### PR TITLE
Feature/run csscomb

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- run csscomb
- updates box-shadow style order

- [x] responsive behaviour tested
- [x] browser and device tested
- [x] passes validation (html / bootstrap)
- [x] SEO optimized (html outline)
- [x] printability (print css)
- [x] doesn’t break anything else

### CODE REVIEW

**Comment by Brankub** [comment]
:+1:

File: djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss [hunk 1]
```diff
@@ -4,70 +4,100 @@
...
+    clip: rect(0 0 0 0);
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    padding: 0;
+    border: 0;
+    // Extends the .visuallyhidden class to allow the element to be focusable
+    // when navigated to via the keyboard: drupal.org/node/897638
+    &.focusable:active,
+    &.focusable:focus {
```
Comment (dhorelik) [inline djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss]:
```
&.focusable {
    &:active,
    &:focus {
        // styling here
    }
} 
```

File: djangocms_admin_style/sass/libs/_normalize.scss [hunk 2]
```diff
@@ -1,433 +1,440 @@
...
+
+    //
+    // 1. Corrects color not being inherited in IE6/7/8/9
+    // 2. Corrects alignment displayed oddly in IE6/7
+    //
+
+    legend {
+        border: 0; // 1//
+
+        *margin-left: -7px; // 2//
```
Comment (dhorelik) [inline djangocms_admin_style/sass/libs/_normalize.scss]: this needs IE6-7 support, right? haven't seen this in a while =)

File: djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss]: what exactly do you mean?

File: djangocms_admin_style/sass/partials/_content.scss [hunk 3]
```diff
@@ -1,263 +1,591 @@
...
+            color: $font-color !important;
+            font-size: 13px;
+            line-height: 16px; 
+            margin-bottom: 0;
+            border: none;
+            background: none !important;
+        }
+        input {
+            width: 252px;
+            margin: -5px 0 2px 0;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_content.scss]: `margin: -5px 0 2px;`

File: djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss]: ahh sorry ;)
will change this and add it to new pr

File: djangocms_admin_style/sass/partials/_content.scss [hunk 4]
```diff
@@ -1,263 +1,591 @@
...
+                        width: 100%;
+                    }
+                    input {
+                        font-size: 16px;
+                        width: 240px !important; 
+                        padding: 10px;
+                    }
+                }
+                .submit-row {
+                    padding: 20px 0 0 0;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_content.scss]: `padding: 20px 0 0;`

File: djangocms_admin_style/sass/partials/_forms.scss [hunk 5]
```diff
@@ -1,108 +1,251 @@
...
+        vertical-align: middle;
+    }
+    fieldset.collapsed h2 {
+        margin-bottom: 0;
+        padding-bottom: 5px; 
+        border-bottom: none;
+        background-image: none;
+    }
+    label {
+        display: inline-block; /*min-width: 160px;*/
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_forms.scss]: maybe remove the commented out `min-width`?

File: djangocms_admin_style/sass/partials/_content.scss (hunk 4 above)
Comment (lory87) [inline djangocms_admin_style/sass/partials/_content.scss]: will update this in next pr

File: djangocms_admin_style/sass/partials/_forms.scss [hunk 6]
```diff
@@ -1,108 +1,251 @@
...
+    }
+}
+div.calendarbox,
+div.clockbox {
+    margin: 0;
+    padding: 0;
+    border: 1px solid #ccc;
+    border-radius: $default-border-radius;
+    background: #f3f3f3;
+    box-shadow: 0 1px 10px rgba(0,0,0,0.1);
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_forms.scss]: I see csscomb doesn't add space between rgba values, maybe do this manually still? `rgba(0, 0, 0, 0.1)`

File: djangocms_admin_style/sass/partials/_iconsOLD.scss [hunk 7]
```diff
@@ -35,58 +33,116 @@ a.active.selector-chooseall:before,
...
-a[id^="calendarlink"] img, , a[id^="clocklink"] img { display: none;}
+a[id^="calendarlink"],
+a[id^="clocklink"] {
+    display: inline-block;
+    font-size: 19px;
+    width: 19px;
+    height: 19px;
+    padding-left: 0 !important;
+}
+a[id^="calendarlink"] img,,
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_iconsOLD.scss]: 2 commas

File: djangocms_admin_style/sass/partials/_messages.scss [hunk 8]
```diff
@@ -1,264 +1,267 @@
...
+    font-size: 125%;
+    font-weight: bold;
+    text-align: center;
+    width: 100%;
+
+    ul {
+        margin: 0;
+        padding: 0;
+        list-style: none;
+        //border-bottom: 1px solid $line-color-medium;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_messages.scss]: same here regarding the comment. Maybe remove it altogether?

File: djangocms_admin_style/sass/partials/_messages.scss [hunk 9]
```diff
@@ -1,264 +1,267 @@
...
-			box-shadow: 0 0 0 0;
-		}
-	}
+    &.header-info {
+        @include message-info;
+        .meta small {
+            color: #fff;
+        }
+        nav li.internal {
+            box-shadow: 0 0 0 0;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_messages.scss]: `box-shadow: 0;` would do. Or `box-shadow: none;` Same below.

File: djangocms_admin_style/sass/partials/_mixins.scss [hunk 10]
```diff
@@ -1,172 +1,193 @@
...
-	border-radius: $default-border-radius; box-shadow: 0 1px 2px 1px rgba(0,0,0,0.08); padding: 0;
-	background: $background-color-white; border: none; margin-bottom: 5px;
-	h2, h3, h4, h5, h6 { margin-bottom: 12px; margin-top: 0; }
+@mixin box() {
+    margin-bottom: 5px;
+    padding: 0;
+    border: none;
+    border-radius: $default-border-radius;
+    background: $background-color-white;
+    box-shadow: 0 1px 2px 1px rgba(0,0,0,0.08);
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_mixins.scss]: spaces within rgba

**Comment by dhorelik** [comment]
overall lgtm, feel free to address my comments in the next PR

**Pull request merged**